test/integration/crud/crud_api.test.ts

```diff
@@ -304,476 +303,249 @@ describe('CRUD API', function () {
-              { deleteOne: { filter: { c: 1 } } },
-              { deleteMany: { filter: { c: 1 } } },
-              { replaceOne: { filter: { c: 3 }, replacement: { c: 4 }, upsert: true } }
-            ],
-            { ordered: true, writeConcern: { w: 1 } },
-            function (err, r) {
-              // expect(err).to.not.exist;
-              test.equal(1, r.insertedCount);
-              test.equal(2, r.upsertedCount);
-              test.equal(1, r.deletedCount);
-
-              // Crud fields
-              test.equal(1, r.insertedCount);
-              test.equal(1, Object.keys(r.insertedIds).length);
-              test.equal(1, r.matchedCount);
-              test.equal(1, r.deletedCount);
-              test.equal(2, r.upsertedCount);
-              test.equal(2, Object.keys(r.upsertedIds).length);
-
-              client.close(done);
-            }
-          );
-        });
-      };
 
-      legacyInsert();
+      test.equal(3, r.insertedCount);
+      test.equal(1, r.upsertedCount);
+      test.equal(1, r.deletedCount);
+
+      // Crud fields
+      test.equal(3, r.insertedCount);
+      test.equal(3, Object.keys(r.insertedIds).length);
+      test.equal(1, r.matchedCount);
+      test.equal(1, r.deletedCount);
+      test.equal(1, r.upsertedCount);
+      test.equal(1, Object.keys(r.upsertedIds).length);
+    });
+
+    it('bulk write ordered', async function () {
+      const db = client.db();
+      const i = await db.collection('t2_6').insertMany([{ c: 1 }], { writeConcern: { w: 1 } });
+      expect(i).property('insertedCount').to.equal(1);
+
+      const r = await db
+        .collection('t2_6')
+        .bulkWrite(
+          [
+            { insertOne: { document: { a: 1 } } },
+            { insertOne: { document: { g: 1 } } },
+            { insertOne: { document: { g: 2 } } },
+            { updateOne: { filter: { a: 2 }, update: { $set: { a: 2 } }, upsert: true } },
+            { updateMany: { filter: { a: 2 }, update: { $set: { a: 2 } }, upsert: true } },
+            { deleteOne: { filter: { c: 1 } } },
+            { deleteMany: { filter: { c: 1 } } }
+          ],
+          { ordered: true, writeConcern: { w: 1 } }
+        );
+
+      test.equal(3, r.insertedCount);
+      test.equal(1, r.upsertedCount);
+      test.equal(1, r.deletedCount);
+
+      // Crud fields
+      test.equal(3, r.insertedCount);
+      test.equal(3, Object.keys(r.insertedIds).length);
+      test.equal(1, r.matchedCount);
+      test.equal(1, r.deletedCount);
+      test.equal(1, r.upsertedCount);
+      test.equal(1, Object.keys(r.upsertedIds).length);
     });
   });
 
-  it('should correctly execute update methods using crud api', {
-    // Add a tag that our runner can trigger on
-    // in this case we are setting that node needs to be higher than 0.10.X to run
-    metadata: {
-      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
+  describe(
```

Comment:
Same here, please see my comment from the above with the reasoning behind changing single it into describe.